control_plane/src/lib.rs

```diff
@@ -9,4 +9,5 @@
 
 pub mod compute;
 pub mod local_env;
+pub mod remotes;
```

Comment:
should we stick to calling everything origin(s)?

Reply:
If we follow git's convention, we should call them "remotes", and by convention one of the remotes would be called "origin".

Reply:
Fair enough. I always mess this remote/origin when typing git commands and in RFC I also used term origin instead of remote. Let's stick to git convention.